handlers/main.yml

```diff
@@ -1,13 +1,13 @@
 ---
 - name: restart-systemd-buildkite
   systemd:
-    name: "{{ buildkite_agent_username }}"
+    name: "buildkite-agent"
```

Comment:
This is the service-name, not the user-name, so is more properly `buildkite-agent`.